src/components/Pages/OrderPage.tsx

```diff
@@ -26,19 +27,27 @@ const initialFormState: IForm = {
 const OrderPage = () => {
   const [showModal, setShowModal] = useState(false);
   const navigate = useNavigate();
-  const cartContext = useContext(CartContext);
-  const { mutate, error, isError } = useMutation<any, Error, IPost_Query_Form>(
-    postOrderToHistory
-  );
+  const dispatchAction = useDispatch();
+  const error = useSelector((state: any) => state.orders.error);
+
+  const cartTotalAmount = useSelector((state: any) => {
+    return state.cart.totalAmount;
+  });
+
+  const cartContext = useSelector((state: any) => {
+    return state.cart;
+  });
```

Comment:
it's better to keep your selector functions somewhere in another file and export it for convinience / potential reusability etc
any 😢